roles/galaxy-worker/templates/galaxy-worker.deployment.yaml.j2

```diff
@@ -120,6 +120,8 @@ spec:
           args:
           - start-worker
           env:
+            - name: ENTRYPOINT
+              value: "{{ _entrypoint_dir }}/pulpcore-worker"
```

Comment:
I don't think we need this for worker

https://github.com/ansible/galaxy_ng/blob/master/docker/bin/start-worker